doc/api/crypto.md

```diff
@@ -2083,8 +2062,8 @@ added: v10.0.0
 <!-- YAML
 added: v0.9.3
 -->
-* Returns: {string[]} An array of the names of the supported hash algorithms,
-  such as `'RSA-SHA256'`.
+* Returns: {string[]} An array of the names of the supported hash algorithms
+  such as `'RSA-SHA256'`. Hash algorithms also called "digest" algorithms.
```

Comment:
Was this supposed to become a full sentence?

Reply:
yes, it is missing a word, I will fix!